test/removeFromTeam.js

```diff
@@ -47,6 +79,56 @@ describe("removing a user from a team", function() {
         );
     }));
 
-    // TODO: add tests for more hidden groups other than just in announcements
+    it("should remove the user from event hidden groups", coroutine(function*() {
+        let date = new Date();
+        let events = yield sessions[0]("GET",
+            `/events/startYear/${date.getFullYear() - 1}/startMonth/0
+                /endYear/${date.getFullYear() + 1}/endMonth/0`);
```

Comment:
Using backticks for a template string will include the line break and indentation, which is not intended here. Just use 2 strings and a `+`.